### DISCUSSION TITLE
Risk

### DISCUSSION THREAD

**Discussion by operator007forever** [JustOptimize/return-ShowHiddenChannels#184] [Q&A] [answered]
When you use the show hidden channels feature, will anyone be able to detect me?

**Comment by JustOptimize** [comment, marked as answer]
No, the only thing that is "detectable" is that it fetches the users in the "users that can see this channel" but discord normally fetches them so, not really a way to know if you're using it or just clicking on random users

Edit: With [v0.4.9](https://github.com/JustOptimize/ShowHiddenChannels/releases/tag/v0.4.9) the plugin was made even more undetectable adding a random delay between user fetching simulating a user clicking random profiles
